module/renderer/src/commonMain/kotlin/org/cru/godtools/shared/renderer/content/RenderContent.kt

```diff
@@ -19,6 +22,10 @@ internal fun ColumnScope.RenderContent(content: List<Content>, state: State) {
 
 @Composable
 internal fun ColumnScope.RenderContent(content: Content, state: State) {
+    val visible by remember(content, state) { content.isGoneFlow(state) }.collectAsState(false)
+
+    if (!visible) return
```

Comment:
this logic is backwards, `visible` will currently be `true` if the element is gone, or `false` if the element is actually visible.

you should update the variable name to make it more clear